python/python/glide/async_commands/cluster_commands.py

```diff
@@ -367,3 +368,87 @@ async def lastsave(self, route: Optional[Route] = None) -> TClusterResponse[int]
             TClusterResponse[int],
             await self._execute_command(RequestType.LastSave, [], route),
         )
+
+    async def sort(
+        self,
+        key: str,
+        limit: Optional[Limit] = None,
+        order: Optional[OrderBy] = None,
+        alpha: Optional[bool] = None,
+    ) -> List[str]:
```

Comment:
Why not
```suggestion
    ) -> List[Optional[str]]:
```

Reply:
If it is not a mistake, please describe why in docs and update transaction docs too.

Reply:
In CME there is no by/get_pattern (which can lead to get None value in the returned key), meaning we get a sorted list or None in return.